File: Basics/Conditional Statements Advanced/Exercise/01.py
projections = input()
rows = int(input())
columns = int(input())

all_place = rows * columns
price = 0

if projections == "Premiere":
    price = 12
elif projections == "Normal":
    price = 7.50
elif projections == "Discount":
    price = 5
  
total_cost = all_place * price

print(f"{total_cost:.2f} leva")

File: JS-Basics/Conditional Statements Advanced/Exercise/01.js
function cinemaIncome(projectionType, rows, columns) {
    const prices = {
        "Premiere": 12.00,
        "Normal": 7.50,
        "Discount": 5.00
    };

    const totalSeats = rows * columns;

    const totalIncome = totalSeats * prices[projectionType];

    console.log(`${totalIncome.toFixed(2)} leva`);

}
